moveit_core/constraint_samplers/CMakeLists.txt

```diff
@@ -8,22 +8,18 @@ add_library(${MOVEIT_LIB_NAME} SHARED
   src/union_constraint_sampler.cpp
 )
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-
 ament_target_dependencies(${MOVEIT_LIB_NAME}
```

Comment:
remove MOVEIT_LIB_NAME

Reply:
MOVEIT_LIB_NAME is the target we are adding the dependencies for, not a dependency itself.

Reply:
Got it, thanks